automatic_symbol_loading/automatic_symbol_loading.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+
+import gdb
+
+try:
+    from elftools.elf.elffile import ELFFile
+except ModuleNotFoundError:
+    check_build_id = False
+else:
+    check_build_id = True
+
+
+def create_file_dict(path: str) -> dict:
+    ret_dict = {}
+    p = sorted(Path(path).glob("**/*.debug"))
```

Comment:
Why is it sorted?

Reply:
`p` is not a very useful name. What does it stand for?
Or you could just inline it in the `for`.

Reply:
Sorted() is a left over of when I was experimenting, as it adds nothing I can remove it.

I will also remove "p" and inline it in the for